# HG changeset patch
# User Rik <rik@octave.org>
# Date 1447950061 28800
#      Thu Nov 19 08:21:01 2015 -0800
# Branch stable
# Node ID 234589c84d4a629171bfbb6d798cf4d437ecbf62
# Parent  a7903de9c35982bf6c1672d3c7e9eb427354b856
Fix rotated graphics when printing through ghostscript (bug #46435).

* __ghostscript__.m: Add "-dAutoRotatePages=/None" to liste of base ghostscript
options.

diff --git a/scripts/plot/util/private/__ghostscript__.m b/scripts/plot/util/private/__ghostscript__.m
--- a/scripts/plot/util/private/__ghostscript__.m
+++ b/scripts/plot/util/private/__ghostscript__.m
@@ -61,17 +61,17 @@ function [gs_cmd, cleanup_cmd] = __ghost
   endfor
 
   if (isempty (opts.papersize))
     format_for_printer = false;
   else
     format_for_printer = true;
   endif
 
-  gs_opts = ["-dQUIET -dNOPAUSE -dBATCH -dSAFER -sDEVICE=" opts.device];
+  gs_opts = ["-dQUIET -dNOPAUSE -dBATCH -dSAFER -dAutoRotatePages=/None -sDEVICE=" opts.device];
 
   if (! isempty (opts.level) && ismember (opts.level, [1, 2, 3]))
     gs_opts = sprintf ("%s -dLanguageLevel=%d", gs_opts, opts.level);
   endif
 
   if (opts.antialiasing && isempty (strfind (opts.device, "write")))
     ## Apply anti-aliasing to all bitmap formats/devices
     gs_opts = sprintf ("%s -dTextAlphaBits=%d -dGraphicsAlphaBits=%d",
